src/components/Property/PropertyFlags/index.js

```diff
@@ -76,6 +78,8 @@ const TenureDetails = ({
       <Alerts alerts={personAlerts} alertType="Contact" />
     )
 
+  const showBoilerHouseDetails = () => boilerHouseId !== ''
```

Comment:
Would it be worth including a check that the asset type is boilerhousable, as well as the ID being not null?

Reply:
I could, however, I don't think this is necessary. There is already a restriction on MMH when adding a boiler house. to a property